src/BeastSearcher.js

```diff
@@ -1,42 +1,77 @@
 import React from "react";
 import Button from "react-bootstrap/Button"
+import Nav from "react-bootstrap/Nav";
+import Form from "react-bootstrap/Form";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faXmark } from '@fortawesome/free-solid-svg-icons'
 
 class BeastSearcher extends React.Component {
 
-  onSubmit = (event) => {
+  onSubmit = event => {
     event.preventDefault()
     let form = event.target
     let newFilter = form[0].value
     this.props.filterBeasts(newFilter)
     form.reset()
   }
 
+  onHornSelect = key => {
+    let result = null
+    if(key !== 'no-horn') {
+      // in short: "2-horn" => 2
+      result = parseInt(key.substring(0, key.indexOf('-')))
+    }
+    this.props.setHornLevel(result);
+  }
+
   render() {
     const {
       resetBeastFilter,
       removeBeastFilter,
+      filters,
+      byHorns,
     } = this.props
     return (
       <>
-        <form className="searchbar" onSubmit={this.onSubmit}>
-          <p>
-            Search a beast by name!
-          </p>
-          <input type="text" id="searchTerm" name="name" required />
-          <Button variant="primary" type="submit">
-            Submit
-          </Button>
-          <Button disabled={!this.props.filters.length} onClick={() => resetBeastFilter()} variant="warning">
-            Remove filters
-          </Button>
-        </form>
+        <div className="searchbar">
+          <Nav onSelect={this.onHornSelect} justify variant="tabs" defaultActiveKey="no-horn">
+            <Nav.Item>
+              <Nav.Link eventKey="no-horn">All Beasts</Nav.Link>
+            </Nav.Item>
+            <Nav.Item>
+              <Nav.Link eventKey="1-horn">One Horned</Nav.Link>
+            </Nav.Item>
+            <Nav.Item>
+              <Nav.Link eventKey="2-horn">Two Horned</Nav.Link>
+            </Nav.Item>
+            <Nav.Item>
+              <Nav.Link eventKey="3-horn">Three Horned</Nav.Link>
+            </Nav.Item>
+            <Nav.Item>
+              <Nav.Link eventKey="100-horn">100 Horned</Nav.Link>
+            </Nav.Item>
+          </Nav>
+          <Form onSubmit={this.onSubmit}>
+            <Form.Group className="m-3" controlId="formBasicPassword">
+              <Form.Label>Search a beast by name!</Form.Label>
+              <Form.Control type="text" placeholder="Name" />
+              <Form.Text className="text-muted">
+                Will apply on top of filtering by horn count at the top.
+              </Form.Text>
+            </Form.Group>
+            <Button className="mb-3 ml-3" variant="primary" type="submit">
+              Submit
+            </Button>
+            <Button className="mb-3 ml-3" disabled={!filters.length} onClick={() => resetBeastFilter()} variant="warning">
```

Comment:
I would choose more semantic class names. mb-3 isn't super descriptive. As someone who didn't write this code I have no idea what this means or is about. And the thing is, in about two weeks you'll also read this code like someone who didn't write it. Keep your future self in mind.